CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
+* AKS and Holiday Season: To ease the burden of upgrade and change during the holiday season, AKS is extending a limited scope of support for all clusters and node pools on 1.19 as a courtesy. Customers with clusters and node pools on 1.19 after the [announced deprecation date of 2021-11-30](https://docs.microsoft.com/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar) will be granted an extension of capabilities outside the [usual scope of support for deprecated versions](https://docs.microsoft.com/azure/aks/supported-kubernetes-versions#kubernetes-version-support-policy).
+ The scope of this limited extension is effective from '2021-12-01 to 2022-01-31' and is limited to the following:
+  * Creation of new clusters and node pools on 1.19.
+  * CRUD operations on 1.19 clusters.
+  * Azure Support of non-Kubernetes related, platform issues. Platform issues include trouble with networking, storage, or compute running on Azure. Any support requests for K8s patching and troubleshooting will be requested to upgrade into a supported version.
+
+### Release Notes
+
+* New Features
+  * Windows containerd GA with Containerd version 1.5.8. Containerd will be used by default in windows node with k8s version 1.22.3 or newer, otherwise, Docker will be used. Docker  will be deprecated in AKS for the k8s version newer than 1.23.0. To enable Containerd under k8s version 1.22.3, the customer header WindowsContainerRuntime=containerd is required during upgrade or creation. To enable Docker with k8s newer than 1.22.3 and older than 1.23.0, the customer is required to set the customer header WindowsContainerRuntime=docker. NOTE: Containerd cannot be reverted to Docker.
+* Preview Features
+  * Node pool start/stop is now in preview. With this feature you are able to stop and start your node pool whenever you see fit. Read more [here](https://docs.microsoft.com/azure/aks/start-stop-nodepools).
+* Bug Fixes
+  * The DelegateFSGroupToCSIDriver feature flag is only introduced to kubelet in 1.22, the nodes on 1.21 wont be able to start with this feature flag.
+  * A WindowsGmsaProfile certificate renewal issue during certificate rotation has been identified and fixed.
+  * Added the component=tunnel label to konnectivity-agent pods so they will be matched by any label selectors that previously matched tunnelfront pods. This only applies to clusters that have received the new Konnectivity network tunnel.
+* Component Updates
+  * When cluster has Bring Your Own Key (BYOK) setting, and os disk is ephemeral by default then the fix is to ignore BYOK os disk setting and set os disk as ephemeral. Users could still use BYOK for data disk feature.
+  * AKS Ubuntu 18.04 image updated to [AKSUbuntu-1804-2021.11.27](vhd-notes/aks-ubuntu/AKSUbuntu-1804/2021.11.27.txt) - please refer to the link for package versions in this VHD.
+  * Increased cpu limits of csi driver node daemonsets to prevent cpu throttling.
```

Comment:
Increased to what?

Reply:
Also this is Behavior change.

Reply:
Should we expose the increase limit to end customer ?

Reply:
I think it would be good to see here. Will help with capacity planning. Does this take it over the 20% per node?

Reply:
THis is now missing

Reply:
Added it with the CPU numbers.